install_config/customizing-configurations-in-the-tektonconfig-cr.adoc

```diff
@@ -13,7 +13,7 @@ In {pipelines-title}, you can customize the following configurations by using th
 * Changing the default service account
 * Disabling the service monitor
 * Configuring pipeline resolvers
-* Disabling cluster tasks and pipeline templates
+* Disabling pipeline templates
```

Comment:
We have this in the docs:
```
You can set the value of the pipelinesTemplates parameter to true only when the value of the clusterTasks parameter is true.
```
Does that mean that we should remove mentions of pipeline templates as well? Or will it work without cluster tasks?

Reply:
@ppitonak I agree, we need to update the templates to use resolvers.
cc @savitaashture

Reply:
Okay so from the conversation ,I will keep it there :+1: I will remove the line that talks about its dependency on cluster tasks.

Reply:
@vdemeester @savitaashture we not only should update templates, we have to
```
$ oc edit tektonconfig config
error: tektonconfigs.operator.tekton.dev "config" could not be patched: admission webhook "validation.webhook.operator.tekton.dev" denied the request: validation failed: pipelineTemplates cannot be true if clusterTask is false: spec.addon.params
```

Reply:
@ppitonak @vdemeester This case is handled in Shiv's PR https://github.com/tektoncd/operator/pull/2376

pipeline templates now use ecosystem tasks using resolver  https://github.com/tektoncd/operator/blob/main/cmd/openshift/operator/kodata/tekton-addon/tekton-pipeline-template/pipeline_using_workspace.yaml#L26-L32

Handled this case as well `If resolverTask is false pipelineTemplates cannot be true`

Reply:
> You can set the value of the pipelinesTemplates parameter to
@eromanova97 it can be changed to like this now

```
You can set the value of the pipelinesTemplates parameter to true only when the value of the resolverTask parameter is true.
```